packages/components/dropdown/src/dropdown.ts

```diff
@@ -28,59 +28,56 @@ export interface IElDropdownInstance {
   triggerElm?: ComputedRef<Nullable<HTMLButtonElement>>
 }
 
-export const dropdownProps = {
+export const dropdownProps = buildProps({
```

Comment:
The reason that I didn't add this method here is that when extending from other component props, `buildProps` tend to complain about the type not matching since the props that we were using is built by `buildProps`.